src/scheduler.h

```diff
@@ -46,15 +46,19 @@ class CScheduler
     // Call func at/after time t
     void schedule(Function f, std::chrono::system_clock::time_point t);
 
-    // Convenience method: call f once deltaMilliSeconds from now
-    void scheduleFromNow(Function f, int64_t deltaMilliSeconds);
+    /** Call f once after the delta has passed */
+    void scheduleFromNow(Function f, std::chrono::milliseconds delta)
+    {
+        schedule(std::move(f), std::chrono::system_clock::now() + delta);
+    }
 
-    // Another convenience method: call f approximately
-    // every deltaMilliSeconds forever, starting deltaMilliSeconds from now.
-    // To be more precise: every time f is finished, it
-    // is rescheduled to run deltaMilliSeconds later. If you
-    // need more accurate scheduling, don't use this method.
-    void scheduleEvery(Function f, int64_t deltaMilliSeconds);
+    /**
+     * Repeat f until the scheduler is stopped. First run is after delta has passed once.
+     *
+     * The timing is not exact: Every time f is finished, it is rescheduled to run again after delta. If you need more
+     * accurate scheduling, don't use this method.
+     */
```

Comment:
suggestion:
```cpp
    /**
     * Repeat f indefinitely until the scheduler is stopped. The first run
     * occurs after delta time.
     *
     * The timing is not exact; every time f finishes, it is scheduled again
     * to run delta time later. If you need more accurate scheduling, don't
     * use this method.
     */
```

Reply:
might take, if I have to force push for other reasons